### DISCUSSION TITLE
Command key on macOS

### DISCUSSION THREAD

**Discussion by mskvsk** [sxyazi/yazi#1056] [1. Q&A]
Is there a way to make a key binding with the Command key on macOS?

For example, Neovim uses D- notation but it doesn't seem to work in Yazi.

**Comment by sxyazi** [comment]
Not supported atm, but we can add it. Would you like to create a feature request for it?

**Reply by mskvsk** [reply]
Thank you, will do!

**Comment by sxyazi** [comment]
For someone passing by, it's already supported in the latest main branch, https://github.com/sxyazi/yazi/issues/1065
